.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -48,10 +48,10 @@ body:
       label: Version
       description: What is the version of CloudNativePG you are running?
       options:
-        - "1.24.0"
-        - "1.23.4"
+        - "1.25 (latest patch)"
+        - "1.24 (latest patch)"
```

Comment:
the `latest patch` is not immediately clear to me.
I guess it means "the latest patch in 1.2x", but, I read it initially as
"1.25 is the latest patch".